packages/grafana-ui/src/components/Forms/Legacy/Switch/Switch.story.internal.tsx

```diff
@@ -11,22 +12,26 @@ const getStory = (title: string, component: any) => ({
     docs: {
       page: mdx,
     },
+    knobs: {
+      disable: true,
+    },
+  },
+  argTypes: {
+    className: NOOP_CONTROL,
+    labelClass: NOOP_CONTROL,
+    switchClass: NOOP_CONTROL,
   },
 });
 
 export default getStory('Forms/Legacy/Switch', Switch);
 
-const getKnobs = () => {
-  return {
-    label: text('Label Text', 'Label'),
-    tooltip: text('Tooltip', ''),
-  };
-};
-
-const SwitchWrapper = () => {
-  const { label, tooltip } = getKnobs();
+const SwitchWrapper: Story<Props> = (args) => {
```

Comment:
```suggestion
const SwitchWrapper: Story<Props> = ({label, ...args}) => {
```
You can also destructure argument and separate the `label` from the rest of the args.